arviz/tests/test_diagnostics.py

```diff
@@ -433,12 +459,12 @@ def test_multichain_summary_array(self, draws, chains):
                 )
             ).all()
         else:
-            assert mcse_mean_hat == mcse_mean_hat_
-            assert mcse_sd_hat == mcse_sd_hat_
-            assert ess_mean_hat == ess_mean_hat_
-            assert ess_sd_hat == ess_sd_hat_
-            assert ess_bulk_hat == ess_bulk_hat_
-            assert ess_tail_hat == ess_tail_hat_
+            assert_almost_equal(mcse_mean_hat, mcse_mean_hat_)
+            assert_almost_equal(mcse_sd_hat, mcse_sd_hat_)
+            assert_almost_equal(ess_mean_hat, ess_mean_hat_)
+            assert_almost_equal(ess_sd_hat, ess_sd_hat_)
+            assert_almost_equal(ess_bulk_hat, ess_bulk_hat_)
+            assert_almost_equal(ess_tail_hat, ess_tail_hat_)
```

Comment:
Is there some difference between the results?

Reply:
one of the results was the rounded version of the other one. This rounding off took place at around 10th or 11th place of the decimal iirc.

Reply:
Can you check the summary code and see if it needs update? (It's is basically copy from the main functions, just trying to minimize duplicate calculations)